recipe/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set name = "scrubadub" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LeapBeyond/scrubadub/archive/v{{ version }}.tar.gz
+  sha256: 43652270d9e808318bf1741ddeb08a9dea30a91d28f3a3963041f30be9fc8d25
+  patches:
+    - patches/0001-changed-textblob-pin.patch
+    - patches/0002-disable-pep-warnings.patch
+    - patches/0003-fix-asserts.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    # The conda main + snowflake channels have only 0.17.1 and 0.18.0.post0 versions.
+    # https://github.com/LeapBeyond/scrubadub/blob/v2.0.1/requirements/python#L2 
+    # upstream textblob==0.15.3
+    - textblob <=0.18.0.post0
```

I would change this to
```suggestion
    - textblob >=0.15.3
```